src/doc/trpl/error-handling.md

```diff
@@ -279,7 +279,7 @@ fn extension(file_name: &str) -> Option<&str> {
 }
 ```
 
-One other pattern that we find is very common is assigning a default value to
```

Comment:
Thank you @Manishearth.

I understand.
I think that the sentence @caipre suggested is easier to read.
How do you think about that?